Documentation/contributing/development/debugging.rst

```diff
@@ -4,10 +4,69 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _gs_debugging:
+
 #########
 Debugging
 #########
 
+Attaching a debugger
+--------------------
+
+Cilium comes with a handy set of Makefile targets for quickly deploying
+development builds to a local :ref:`Kind <gs_kind>` cluster. One of them is
+``kind-debug-agent``, generating a container image that wraps the Cilium agent
+with a `dlv <https://github.com/go-delve/delve>`_ invocation, causing the agent
+process to listen for connections from a debugger frontend on port 2345.
```

Comment:
Can we make some changes to remove the indirect references in here? For instance:

> One of these Makefile targets is named ``kind-debug-agent``, which generates a container image that wraps the Cilium agent with a dlv invocation. This enables the Cilium agent to listen for connections from a debugger on port 2345.

Reply:
What are indirect references?

Reply:
Indirect references are using words like 'them' and 'it' to refer to something. For instance, "One of them" is an indirect reference to a Makefile target. The change you made looks great!